maps/glloydstation/glloydstation.dm

```diff
@@ -1,5 +1,8 @@
 #if !defined(USING_MAP_DATUM)
 
+	#include "../../code/modules/urist/area&turf/areas.dm"
+	#include "../../code/modules/urist/machinery/misc.dm"
```

Comment:
Relative pathing is bad practice, better to just move these files wholesale to shared/urist or something along these lines